test/reporter-proxy.test.js

```diff
@@ -109,6 +109,11 @@ describe('reporterProxy', function() {
       assert.deepEqual(addTimingArgs[2], {gitHubPackageVersion: version});
 
       assert.deepEqual(incrementCounterStub.lastCall.args, [counterName]);
+
+      assert.deepEqual(reporterProxy.events.length, 0);
```

Comment:
For future reference: you can do `assert.lengthOf(thing, 0)` to get a slightly more informative error message when it fails. :stars:

Reply:
oh, thanks!  I'll clean this up.

Reply:
Haha that wasn't meant as "you have to fix this"! It was just a random note because I'm tired :eyes:

Reply:
totally, didn't think you meant it that way, just want to get in the habit of using better assertions <3